iphone/Maps/LocalizedStrings/en.lproj/Localizable.strings

```diff
@@ -1197,7 +1197,7 @@
 
 "transit_not_found" = "Subway navigation in this region is not available yet";
 
-"dialog_pedestrian_route_is_long_header" = "Subway route is not found";
+"dialog_pedestrian_route_is_long_header" = "No subway route found";
```

Comment:
Thanks! This string should be fixed in https://github.com/organicmaps/organicmaps/blob/master/data/strings/strings.txt

Reply:
@organicmaps/translations-en-gb WDYT? `No metro route found` sounds ok too?

Reply:
Some cities have subways, some have metros.